src/Tester/StreamingTests/PullingAgentManagementTests.cs

```diff
@@ -16,30 +16,23 @@ namespace UnitTests.StreamingTests
     [DeploymentItem("ClientConfigurationForStreamTesting.xml")]
     [DeploymentItem("OrleansProviders.dll")]
     [TestClass]
-    public class PullingAgentManagementTests : UnitTestSiloHost
+    public class PullingAgentManagementTests : UnitTestSiloHostPerFixture
```

Comment:
yeah, part of this refactoring was also trying to avoid relying "too much" on MSTest-specific features, so that I don't complicate the upcoming migration unnecessarily